_api-reference/index-apis/stats.md

```diff
@@ -644,4 +644,17 @@ GET /testindex*/_stats?expand_wildcards=open,hidden
 ```json
 GET /testindex/_stats?level=shards
 ```
-{% include copy-curl.html %}
\ No newline at end of file
+{% include copy-curl.html %}
+
+## Concurrent segment search
```

Comment:
These stats will also show up in [nodes-stats](https://opensearch.org/docs/2.9/api-reference/nodes-apis/nodes-stats/) as well as `_cat/nodes`, `_cat/shards`, `_cat/indices` but for the `_cat` APIs users can find it in the help section of the API.

Reply:
@kolchfa-aws since all of these stats are gated by the concurrent search feature flag, I was thinking this information should be kept all within the `concurrent-segment-search.md` page too? I don't think we need to add these to the API pages until the feature is GA and the feature flag is removed, but it would be good to have the table below somewhere.

Reply:
Yes, the two options are:
- keep the APIs with the concurrent segment page
- put the APIs in their respective pages. 
It's a matter of opinion, but I think it makes it easier if the API changes are kept in the API pages. Users won't see concurrent-segment-related fields if they don't enable the feature, so I don't think there's any harm in keeping them in the API pages, especially because they are in a separate table. The danger with keeping them in the main page is that someone can just remove the experimental header and not add the API changes in the GA release.

Reply:
> The danger with keeping them in the main page is that someone can just remove the experimental header and not add the API changes in the GA release.

Maybe we can track this is an issue somewhere - perhaps https://github.com/opensearch-project/documentation-website/issues/3662

Reply:
Added a comment so we can track